docs/quick-start/set-up-byconity-dev-env-on-mac.mdx

```diff
@@ -0,0 +1,99 @@
+# ByConity MAC development environment
+
+The easiest way to build a development environment on a MAC is to develop in a Docker container in vscode using the Dev Containers plugin. This way, you don't have to spend a lot of time configuring your environment.。 For more information, see [offical dev](https://github.com/ByConity/ByConity/blob/master/docker/debian/dev-env/README.md)。
+
+# Config docker
+
+For more information, see [docker quick start in Chinese](https://www.ruanyifeng.com/blog/2018/02/docker-tutorial.html) and [docker quick start in English](https://docs.docker.com/get-started/).
+
+In the Chinese docker quick start reference, the small difference is that you cannot use usermod or systemctl in MAC in step 5 and must replace them with the following two commands.
```

Comment:
I don't think the belowing steps are necessary to run docker on Mac. 
For the image part, we may need to consider the Apple M CPU support.